src/steps/declare-plugin-options-schema.js

```diff
@@ -0,0 +1,76 @@
+const queryTypes = [
+  `UserToMediaItemConnection`,
+  `WpContentNodeToEditLockConnectionEdge`,
+  `WPPageInfo`,
+  /* TODO: finish this list */
+]
+
+export function declarePluginOptionsSchema({ Joi }) {
```

Comment:
We wont be able to get a predefined list of types at this point since the WP site owner registers the available types. We do have the plugin options though, so you should be able to get the keys they're currently using :

```
export function declarePluginOptionsSchema({ Joi }, pluginOptions) {
    const queryTypes = Object.keys(pluginOptions.type)
```